apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -2118,19 +2150,26 @@ export class LedgerComponent implements OnInit, OnDestroy {
+                debitCreditTransactions = s.debitCreditTransactions;
             }
         });
 
-        this.entryUniqueNamesForBulkAction.push(
-            ...[
-                ...debitTrx?.filter(f => f.isChecked).map(dt => dt.entryUniqueName),
-                ...creditTrx?.filter(f => f.isChecked).map(ct => ct.entryUniqueName),
-            ]);
+        let entryUniqueNames: string[] = [];
+        if (debitCreditTransactions?.length) {
+            entryUniqueNames = [...debitCreditTransactions?.filter(debitCreditTransaction => debitCreditTransaction.isChecked).map(debitCreditTransaction => debitCreditTransaction.entryUniqueName)];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)